lib/member_page.rb

```diff
@@ -44,7 +44,11 @@ class MemberPage < Scraped::HTML
   end
 
   field :email do
-    email_node.first.to_s.sub('mailto:', '')
+    email_addresses = email_node.map do |node|
+      node.text.sub('mailto:', '')
+    end
```

Comment:
When you introduce a variable to calculate and hold something that isn't what you're actually going to return, that's usually a good sign that that's another instance method looking to escape.

The other pointer to that here is that the variable isn't actually necessary, as this is identical to:

```ruby
  field :email do
    email_node.map do |node|
      node.text.sub('mailto:', '')
    end.sort_by { |e| e.include?('parliament.gov.za') ? -1 : 1 }.join(' ; ')
  end
```

but that's quite ugly, and moving to a `{ … }` block is a bit too long for a single line:

```ruby
  field :email do
    email_node.map { |node| node.text.sub('mailto:', '') }.sort_by { |e| e.include?('parliament.gov.za') ? -1 : 1 }.join(' ; ')
  end
```

So that's usually a sign that it's better to extract at least one additional method somewhere. Here I suspect it's going to be two — one to simply return all the email addresses listed, and one to give them in preferred order — with `field :email` then having no logic other than to semicolon-join the second of those)